distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Array_Like_Helpers.enso

```diff
@@ -0,0 +1,18 @@
+import project.Any.Any
+import project.Data.Array.Array
+import project.Data.Numbers.Integer
+
+## PRIVATE
+    The builtin cannot throw `Illegal_Argument` as that type cannot be a
+    builtin, because they do not support default arguments in constructors.
+    So a wrapper is used which will translate a Java exception into the Enso
+    one. This indirection may be removed once the following story is done:
+    https://www.pivotaltracker.com/n/projects/2539304/stories/183817602
```

Comment:
This docs feels like a historical leftover. Builtins can throw `Illegal_Argument` nowadays, I believe.

Reply:
Gone since 1d87ea7fa3